src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabHookCreator.java

```diff
@@ -17,13 +17,15 @@
 import org.gitlab4j.api.models.ProjectHook;
 import org.gitlab4j.api.models.SystemHook;
 
+import static io.jenkins.plugins.gitlabbranchsource.helpers.GitLabHelper.apiBuilder;
```

Comment:
I was wondering if you have a strong reason why using a static import statement at here.

Reply:
This static method is required. The creation of GitLab Api client needs to judge the credentials, there will be duplicate code